WordPress/Classes/ViewRelated/NUX/SignupUsernameTableViewController.swift

```diff
@@ -0,0 +1,237 @@
+import SVProgressHUD
+
+protocol SignupUsernameTableViewControllerDelegate {
+    func usernameSelected(_ username: String)
+    func newSearchStarted()
+}
+
+class SignupUsernameTableViewController: NUXTableViewController {
+    open var currentUsername: String?
+    open var displayName: String?
+    open var delegate: SignupUsernameTableViewControllerDelegate?
+    private var service: AccountSettingsService?
+    private var suggestions: [String] = []
+    private var isSearching: Bool = false
+    private var selectedCell: UITableViewCell?
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        tableView.register(UINib(nibName: "SiteCreationDomainSearchTableViewCell", bundle: nil), forCellReuseIdentifier: SiteCreationDomainSearchTableViewCell.cellIdentifier)
+        setupBackgroundTapGestureRecognizer()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        WPStyleGuide.configureColors(for: view, andTableView: tableView)
+        tableView.layoutMargins = WPStyleGuide.edgeInsetForLoginTextFields()
```

Comment:
Per the design, the row separators should be aligned to the left:
<img width="387" alt="screen shot 2018-02-25 at 11 11 55 am" src="https://user-images.githubusercontent.com/1816888/36644805-c9104570-1a1c-11e8-8178-721ab3b26dfc.png">